autodoc/admin-api/data/admin-api.lua

```diff
@@ -581,6 +582,9 @@ return {
                 * `reachable`: A boolean value reflecting the state of the
                   database connection. Please note that this flag **does not**
                   reflect the health of the database itself.
+            * `configuration_hash`: The hash of the current configuration. This
+              field is only returned when the Kong node is running in DB-less
+              or data-plane mode.
```

Comment:
```suggestion
              or data-plane mode and a valid config has been loaded.
```

Reply:
Actually the field is returned even if no configuration has been loaded.